### ISSUE TITLE
Test does not work

### ISSUE THREAD

**Issue by GuillaumeDesforges** [coin-or/Rehearse#8]
I copy/paste the test code from here : https://github.com/coin-or/Rehearse/blob/master/Rehearse/tests/testRehearse.cpp

_exemple2_ seems to fail :
```
Solution for x : 1.27156e+20
Solution for y : -8.4771e+19
Solution objvalue = : 4096
Assertion 'fabs(3 - model.getSolutionValue(x)) < 0.00000001' failed, file '/home/arsleust/enpc/s3/rechop/tp/cpp/test.cpp' line '88'.Aborted
```

Does it mean my installation is broken ?

**Comment by Celebio** [comment]
There is definitely a problem. Did `make test` work well?

**Comment by chiyuan-goh** [comment]
`make test` fails as well

```
Test 4 : OK
Solution for x1 : 4
Solution for x2 : 14
Solution objvalue = : 154
Test 5 : OK
Solution for x : 1.27156e+20
Solution for y : -8.4771e+19
Solution objvalue = : 4096
Assertion 'fabs(3 - model.getSolutionValue(x)) < 0.00000001' failed, file 'testRehearse.cpp' line '88'.make[2]: *** [test] Abort trap: 6
```

**Comment by Celebio** [comment]
Should be fixed by 3847e5a3b622825bbaf233c2c888af2a660f11dd
